.prettierignore

```diff
@@ -30,6 +30,7 @@ LICENSE
 yarn.lock
 
 coverage/
+coverage-ts/
```

Comment:
I don't see this path used anywhere in this PR. Does the codecov-action just write to here by default?

Reply:
It's possible it's something that's only on my machine because I was playing around with some other tooling.  I get rid of it and delete the dir locally.